tcf_website/templates/base/navbar.html

```diff
@@ -1,56 +1,52 @@
 {% load static %}
 
+{% block styles %}
+<link rel="stylesheet" href="{% static 'base/navbar.css' %}" />
+{% endblock %}
+
 <header>
-<nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top p-3">
-    <!-- Main logo, search bar, collapse nav items -->
-    <div class="d-flex flex-grow-1 logo-search">
-      <a class="navbar-brand logo" href="/">
-          <img src="{% static 'base/img/logo-orange-small.png' %}" width="30" height="30" alt="tCF">
-      </a>
-      <div class="navbar-collapse collapse">
-      <a class="navbar-brand" href="/">theCourseForum</a>
+  <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
+    <!-- tCF logo -->
+    <a class="navbar-brand tCF" href="/">
+      <img class="logo" src="{% static 'base/img/new_logo.png' %}">
+    </a>
+    <!-- tCF search -->
+    <form class="form-inline flex-grow-1" action="/search" method="get">
+      <div class="input-group flex-grow-1">
+        <!-- TODO plz refactor me -->
```

Comment:
what does this TODO involve?

Reply:
moving the border css out of the html

Reply:
Because it's so specific to that input button, it would be better to leave it seperate from the other css file and not clog it up.